README.md

```diff
@@ -272,3 +272,10 @@ WebSocket SubProtocol header.
 If no appropriate auth manager is configured or can be automatically determined
 then the fallback is the NoopAuthManager, which does no external validation
 calls and simply accepts any provided token or credentials.
+
+## JMX AUTHORIZATION HANDLING
```

Comment:
Let's rename this heading `INCOMING JMX CONNECTION AUTHENTICATION` or similar. LGTM otherwise!

Reply:
Ok, done!